src/pybind/mgr/mgr_module.py

```diff
@@ -2328,7 +2328,7 @@ def send_rgwadmin_command(self, args: List[str],
                 cmd,
                 stdout=subprocess.PIPE,
                 stderr=subprocess.PIPE,
-                timeout=10,
+                timeout=60,
```

Comment:
do we need to make this change? probably its good to test this  in an environment other than the vm that we are using to completely make this change? i prefer this to be done in a different PR with a bit more discussion

Reply:
Can be discussed..right now for testing we are doing this so we donot have to change this manually everytime

Reply:
yup. when we are about to merge this, let's remove it!

Reply:
Sure